pre-index.php

```diff
@@ -448,7 +448,7 @@ class="onoffswitch-checkbox" checked>
             }
             ?>
             <?php
-            if (!$noSearchLocation || !$noStartMe || !$noStartLast || !$noFollowMe || !$noPokestops || !$noScannedLocations || !$noSpawnPoints || !$noRanges) {
+            if (!$noSearchLocation || !$noStartMe || !$noStartLast || !$noFollowMe || !$noPokestops || !$noScannedLocations || !$noSpawnPoints || !$noRanges || !$noWeatherOverlay) {
```

Comment:
Good find! It seems that `$noSpawnArea` is also missing, can you add that in your PR?